app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -92,33 +104,145 @@ function ReviewIssuesStep({
       return <BarcodeAttentionWarning onDismiss={onWarningPageDismissed} pii={pii} />;
     }
 
-    return (
-      <Warning
-        heading={t('errors.doc_auth.throttled_heading')}
-        actionText={t('idv.failure.button.warning')}
-        actionOnClick={onWarningPageDismissed}
-        location="doc_auth_review_issues"
-        remainingAttempts={remainingAttempts}
-        troubleshootingOptions={
-          <DocumentCaptureTroubleshootingOptions
-            location="post_submission_warning"
-            showAlternativeProofingOptions={!isFailedResult}
-            heading={t('components.troubleshooting_options.ipp_heading')}
-          />
-        }
-      >
-        {!!unknownFieldErrors &&
-          unknownFieldErrors
-            .filter((error) => !['front', 'back'].includes(error.field!))
-            .map(({ error }) => <p key={error.message}>{error.message}</p>)}
-
-        {remainingAttempts <= DISPLAY_ATTEMPTS && (
-          <p>
-            <strong>{t('idv.failure.attempts', { count: remainingAttempts })}</strong>
-          </p>
-        )}
-      </Warning>
-    );
+    if (!inPersonCtaVariantTestingEnabled || !inPersonURL || isFailedResult) {
+      return (
+        <Warning
+          heading={t('errors.doc_auth.throttled_heading')}
+          actionText={t('idv.failure.button.warning')}
+          actionOnClick={onWarningPageDismissed}
+          location="doc_auth_review_issues"
+          remainingAttempts={remainingAttempts}
+          troubleshootingOptions={
+            <DocumentCaptureTroubleshootingOptions
+              location="post_submission_warning"
+              showAlternativeProofingOptions={!isFailedResult}
+              heading={t('components.troubleshooting_options.ipp_heading')}
+            />
+          }
+        >
+          {!!unknownFieldErrors &&
+            unknownFieldErrors
+              .filter((error) => !['front', 'back'].includes(error.field!))
+              .map(({ error }) => <p key={error.message}>{error.message}</p>)}
+
+          {remainingAttempts <= DISPLAY_ATTEMPTS && (
+            <p>
+              <strong>{t('idv.failure.attempts', { count: remainingAttempts })}</strong>
+            </p>
+          )}
+        </Warning>
+      );
+    }
+    if (inPersonCtaVariantActive === 'in_person_variant_a') {
+      trackEvent('IdV: IPP CTA Variant A');
```

Comment:
This is a side effect in the render path. We'll probably need to move this into a `useEffect` callback.

https://beta.reactjs.org/learn/keeping-components-pure#side-effects-unintended-consequences

Reply:
Same applies to the `trackEvent` usages in the other variants.

Reply:
done in 67718f8f42e740a6e31c09a7edd781f56b26cbd6